utils/webpack/webpack.config.js

```diff
@@ -0,0 +1,207 @@
+const path = require('path');
+const webpack = require('webpack');
+const chalk = require('chalk');
+const TerserPlugin = require('terser-webpack-plugin');
+const HTMLWebpackPlugin = require('html-webpack-plugin');
+const ProgressBarWebpackPlugin = require('progress-bar-webpack-plugin');
+const ShopgateIndexerPlugin = require('./plugins/ShopgateIndexerPlugin');
+const { ENV, isDev, PUBLIC_FOLDER } = require('./lib/variables');
+const getAppSettings = require('./lib/getAppSettings');
+const convertLanguageToISO = require('./lib/convertLanguageToISO');
+const getComponentsSettings = require('./lib/getComponentsSettings');
+const getThemeConfig = require('./lib/getThemeConfig');
+const getThemeLanguage = require('./lib/getThemeLanguage');
+const getDevConfig = require('./lib/getDevConfig');
+const i18n = require('./lib/i18n');
+const getExtensionsNodeModulesPaths = require('./lib/getExtensionsNodeModulesPaths');
+
+const themePath = process.cwd();
+const appConfig = getAppSettings(themePath);
+const componentsConfig = getComponentsSettings(themePath);
+const themeConfig = getThemeConfig(themePath, appConfig);
+const isoLang = convertLanguageToISO(appConfig.language);
+const { sourceMap, ip, apiPort } = getDevConfig();
+const t = i18n(__filename);
+
+const config = {
+  mode: ENV,
+  entry: {
+    app: [
+      path.resolve(__dirname, 'lib', 'polyfill.js'),
+      `${themePath}/index.jsx`,
+    ],
+    common: [
```

Comment:
I see that the module list here was reduced a little bit. Is there is reason for that?

Reply:
Added all the missing modules.